neonvm-vxlan-controller/cmd/main.go

```diff
@@ -164,9 +164,10 @@ func createVxlanInterface(name string, vxlanID int, ownIP string, bridgeName str
 		LinkAttrs: netlink.LinkAttrs{
 			Name: name,
 		},
-		VxlanId: vxlanID,
-		SrcAddr: net.ParseIP(ownIP),
-		Port:    4789,
+		VxlanId:  vxlanID,
+		SrcAddr:  net.ParseIP(ownIP),
+		Port:     4789,
+		Learning: true,
```

Comment:
(question, not-blocking): Should it be passed as an cli argument to the binary? That way only a redeployment would need to be done if we need to rollback the change.

Reply:
I don't think there is a need to rollback this change quickly, so I'll revert this PR if needed.